zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/UserTaskQueryController.java

```diff
@@ -60,4 +63,21 @@ private ResponseEntity<Object> search(final UserTaskQuery query) {
       return RestErrorMapper.mapProblemToResponse(problemDetail);
     }
   }
+
+  @GetMapping(
+      path = "/{userTaskKey}",
+      produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_PROBLEM_JSON_VALUE})
+  public ResponseEntity<?> getByKey(@PathVariable("userTaskKey") final Long userTaskKey) {
+    try {
+      // Success case: Return the left side with the UserTaskItem wrapped in ResponseEntity
+      return ResponseEntity.ok()
+          .contentType(new MediaType(MediaType.APPLICATION_JSON, StandardCharsets.UTF_8))
```

Comment:
❓ Just saw this now, sorry. As far as I can see, the other endpoints don't define the content type explicitly. Do we need to do this? Just wondering.

Reply:
We dont need to define this explicity
I accidently commited this with my copilot 
Will remove it

Thank you for checking this